### DISCUSSION TITLE
Black displays?? What am I missing

### DISCUSSION THREAD

**Discussion by riveroalbert** [flybywiresim/aircraft#3874] [Help] [answered]
EDIT: Update MFS2020 and problem should be gone. 

Sorry I'm sure its stupid but now on Custom FBW be09953 I cannot get any of the display panels to work, or the autopilot panels. They appear black, tried turning the brightness up no luck. Could it have something to do with the recent MFS2020 patches? I tried removing all other mods from the community folder too... What am I missing?
I remember a similar thing happened back in september 2020
Thanks

**Comment by CJORDAN371** [comment]
after the update mine also. nothing in the community folder other than the A32NX folder. It was fine before the update.

**Reply by riveroalbert** [reply]
Tried switching to standard v0.5.3 and same issue. I'm guessing Mfs2020 developers messed something up in the patch, or changed some settings,...

**Comment by PaulFurbacher** [comment]
I have the same problem. With the stable version, all glass cockpit screens and the FMS were black. Changing to the dev build, only the MFD wasn't working.

**Reply by Paytheon** [reply]
I just received the MSFS Update, reinstalled Custom FBW and the panels are working.

**Reply by riveroalbert** [reply]
Thanks I'll give it a try

**Reply by PaulFurbacher** [reply]
I had success, the updater recognized a new version and after downloading it things are fixed.

**Reply by ata68** [reply]
In my case A32nx all screens are black after update 1.14.5.0 MSFS.

What you mean with "reinstalled Custom FBW" ? Can't fix the problem at the moment.

**Reply by danilcek** [reply]
What is "custom FBW"?

**Comment by hotshotp** [comment, marked as answer]
solved in dev and experimental

**Reply by Paytheon** [reply]
I can tell you this black display issue happened with the payware Carendado CT182T Skylane. Once I went to the 'CONTENT MANAGER' and updated everything the black screen issue disappeared. Try going to the 'CONTENT MANAGER' under Profile and update the rest of the content in MSFS 2020. Worked for me.

**Reply by Neirius** [reply]
> solved in dev and experimental

However the stable version has the black screen issue still

**Reply by CJORDAN371** [reply]
I still have black screen for everything but experimental.

**Reply by CptBayer** [reply]
I am doeing my first leg aftr FS update. All screens are fine. Only the VOR function on captains side is not available anymore. I think it depends on the update, cause MS implemented the missing VOR/ADF thing .... omg.... a standart for flying coming 6 month after release .... 


greets Klaus

**Reply by CJORDAN371** [reply]
I have updated, the only thing I haven't done is re-install MSFS2020. still only the experimental version and the MS2020 version work.

**Reply by CJORDAN371** [reply]
Well I just tried a flight with the MSFS2020 B78X, no mods and it is also the same. everything is black, so it must be the game itself.
